rootfs-builder/rootfs.sh

```diff
@@ -95,6 +101,15 @@ AGENT_INIT          When set to "yes", use ${AGENT_BIN} as init process in place
                     of systemd.
                     Default value: no
 
+RUST_AGENT          When set to "yes", build kata-agent from kata-rust-agent instead of go agent
+					Default value: "no"
+
+RUST_AGENT_PKG      URL of the Git repository hosting the agent package.
+                    Default value: ${RUST_AGENT_PKG}
```

Comment:
this `Default` looks differently indented from the ones above/below it - pls check.

Reply:
> @yyyeerbo - btw, I tried running with the latest agent in an initrd and it's not working for me: the agent starts, creates the grpc server and start listening, but the runtime times out trying to talk to the server:
> 
> ```shell
> $ sudo docker run --runtime kata-runtime -ti busybox sh
> docker: Error response from daemon: OCI runtime create failed: Failed to check if grpc server is working: rpc error: code = Unavailable desc = transport is closing: unknown.
> ```
> 
> Could you verify it works for you? If not, I'll debug on Monday probably now...

It worked as @lifupan tested. I guess @lifupan don't test it for a while..
This looks like we didn't set use_vsock in configuration or we are using fc, the first request check is not received..